src/services/date-utilities.ts

```diff
@@ -329,25 +329,25 @@ export default class DateUtilities {
   }
 
   /**
-   * Returns the number of days that 'date' is from today in Pacific timezone.
+   * Returns the number of days that 'date' is from 'initialDate' in Pacific timezone.
    * @returns -1 for yesterday
    * @returns 0 for today
    * @returns +1 for tomorrow
    * @returns NaN in case of error
    */
-  static daysFromToday (today: Date, date: Date): number {
+  static daysBetweenTwoDates (initialDate: Date, date: Date): number {
```

Comment:
I changed this functions name because it wasn't doing what the name was implying.

Reply:
Oh, I see.

Reply:
I think the previous name was OK.

But your name is better. Go ahead and keep it and update the references. I'll close my other comments on this topic.